battan40/KittensToWitches
# Feature/refactor router

### Is this a fix or a feature?
* Feature

### What is the change or fix?
* Adds practical implementation of Router
* Shows the difference between implementing router for component with no functional rendering and 
* implementing component with functionality in their render
* Adds two new components (initially as tests to see if I could connect the router paths)
* One is for a 404 error handle
* The other is the beginning of an about page

### How should this be tested?
* npm start to get the app open
* then go to top of local host address and type /favorites
* or type /about
* or try to go someplace that does not exist (to verify 404 exists)


### Linked Issues:
* closes #12 closes #9 
* #25 #26


There is a potential need for an additional component for favorite cards here.
